cromwell-helm/templates/cromwell.yaml

```diff
@@ -123,18 +123,12 @@ data:
             // This is the maximum polling interval (in seconds):
             maximum-polling-interval = 600
 
-            // Optional Dockerhub Credentials. Can be used to access private docker images.
-            dockerhub {
-              // account = ""
-              // token = ""
-            }
-
             genomics {
               // A reference to an auth defined in the `google` stanza at the top.  This auth is used to create
               // Pipelines and manipulate auth JSONs.
               auth = "application-default"
               // Endpoint for API (dependent on 'location')
-              endpoint-url = "{{ .Values.config.papiv2EndpointUrl }}"
+              endpoint-url = "https://lifesciences.googleapis.com/"
```

Comment:
is the endpoint url different for papi vs. batch? ie. should this be dependent on the value from Leo?

Reply:
Batch does not seem to need this endpoint, it is only required in the papi config

Reply:
ah gotcha, thanks for the clarification!